tox.ini

```diff
@@ -10,5 +10,8 @@ deps =
   -chttps://opendev.org/openstack/requirements/raw/branch/stable/xena/upper-constraints.txt
   -r{toxinidir}/test-requirements.txt
 commands =
+    bash -c "find {toxinidir} -not -path {toxinidir}/.tox/\* \
+    -name \*.rst -type f -print0 | xargs -0 rstcheck"
     sphinx-build --keep-going -W -T -b html -D language=en doc _build/html
     sphinx-build --keep-going -W -b linkcheck doc _build/html
+    pyspelling -c .spellcheck.yml
```

Comment:
tox complains to me to add bash to whitelist_externals

Reply:
`whitelist_externals` were deprecated in [tox 4](https://github.com/tox-dev/tox/issues/1127) in favor to `allowlist_externals`